examples/2_counter/Main.re

```diff
@@ -0,0 +1,24 @@
+open ReWeb;
+
+/* Create a mutable let binding */
+let count = ref(0);
+
+/* Grab the value from count and send it as a text response */
+let getCount = _ => string_of_int(count^) |> Response.of_text |> Lwt.return;
+
+/* Increment the count and then respond with a status of OK */
+let increment = _ => {
+  count := count^ + 1;
```

Comment:
The standard library has a function `incr` that does exactly this, but maybe we should keep the explicit version since this is an example...

Reply:
Yeah I think it's easier to understand this way, especially for beginners.